pristine-zen-garden.html

```diff
@@ -156,7 +147,6 @@ <h3 class="archives">Archives:</h3>
                         </ul>
                     </nav>
                 </div>
-
                 <div class="zen-resources" id="zen-resources">
                     <h3 class="resources">Resources:</h3>
                     <ul>
```

Comment:
This list also has extra spaces inside the li tags

Reply:
@stanley-roache Thank buddy, should be looking good now!
Thoughts?
Good to merge?

Reply:
Looks good!